File: src/main/java/guru/xgm/language/angelscript/psi/tree/AngelScriptIToken.java
package guru.xgm.language.angelscript.psi.tree;

import com.intellij.psi.tree.IElementType;
import guru.xgm.language.angelscript.lang.AngelScriptLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class AngelScriptIToken extends IElementType {

    public AngelScriptIToken(@NotNull @NonNls String debugName) {
        super(debugName, AngelScriptLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "AngelScriptIToken." + super.toString();
    }
}

File: src/main/kotlin/raft/war/language/angelscript/psi/tree/AngelScriptIToken.kt
package raft.war.language.angelscript.psi.tree

import com.intellij.psi.tree.IElementType
import raft.war.language.angelscript.lang.AngelScriptLanguage.Companion.instance
import org.jetbrains.annotations.NonNls

class AngelScriptIToken(debugName: @NonNls String) : IElementType(debugName, instance) {
    override fun toString(): String = "AngelScriptIToken." + super.toString()
}
